src/containers/Chat/ChatConversations/SearchBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { InputBase, Divider, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import SearchIcon from '@material-ui/icons/Search';
+import styles from './SearchBar.module.css';
+
+// /* <SearchBar handleSubmit={handleSearch} isOpen={searchOpen} onReset={resetSearch} defaultVal={searchVal}/> */
+export interface SearchBarProps {
+  handleSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+  searchVal: string;
+  onReset: () => void;
+}
+
+export const SearchBar: React.SFC<SearchBarProps> = (props) => {
```

Comment:
Same code as in TagList, will remove TagList code and use this component for generalization instead after it is fully functional.